rust/relay/server/src/server.rs

```diff
@@ -708,6 +708,15 @@ where
                 (sender, requested_channel),
             );
 
+            // Also update the eBPF maps when refreshing a channel binding
+            self.pending_commands
```

Comment:
I noticed we failed to do this - I'm not sure if it's possible for a client to request different channels when the binding is refreshed? If so, this might fix a bug there.

Reply:
Nah, you can only refresh a channel that you've bound. The functionality is idempotent so this is unnecessary.

A refresh is simply a CHANNEL_BIND of an existing channel.